### DISCUSSION TITLE
FeelsLike parameter

### DISCUSSION THREAD

**Discussion by chpyoung** [cytech/Home-Assistant-wundergroundpws#156] [Q&A] [answered]
Hi,
is possible to grab the FeelsLike information from WU ?
Thank you!
Chris

**Comment by cytech** [comment, marked as answer]
there is no specific "feelslike" returned from the api.
you can use "heatIndex" and "windChill" sensors (current observations)

**Reply by chpyoung** [reply]
OK, thank you!!

**Reply by chpyoung** [reply]
what I was looking into is wind chill.  :)

This is from WU API documentation:
Wind Chill - An apparent temperature. It represents what the air temperature “feels like” on exposed human skin due to the combined effect of the cold temperatures and wind speed.
When the temperature is 61°F or lower the Feels Like value represents the computed Wind Chill so display the Wind Chill value.

Thanks!!

**Comment by bencollinz** [comment]
> Hi, is possible to grab the FeelsLike information from WU ? Thank you! Chris

I pull feels like from openweathermap. works just fine.

**Reply by chpyoung** [reply]
will do it. Thanks!
